src/main/java/com/minecolonies/inventory/GuiField.java

```diff
@@ -0,0 +1,83 @@
+package com.minecolonies.inventory;
+
+import com.minecolonies.colony.ColonyManager;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.colony.ColonyManager'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)